### ISSUE TITLE
React Native - IOS không mở được app test

### ISSUE THREAD

**Issue by shayminmyn** [momo-wallet/mobile-sdk#66]
Cho em hỏi lỗi "could not find method compile() for arguments [project ':react-native-momosdk']" khi chạy demo fix như nào ạ?

**Comment by shayminmyn** [comment]
update lại đã chạy được trên android, implement vào code của em đã chạy được android, nhưng ios thì luôn mở appstore chứ không mở app test ( đã tải app test mới nhất )

**Comment by tuetran0795** [comment]
`
<key>LSApplicationQueriesSchemes</key>
<array>
	<string>momo</string>
</array>
`

bạn xem thử đã add queris vào info.plist chưa nha

**Comment by trancongdung12** [comment]
@tuetran0795 cho mình hỏi có tải được app momo test trên simulator IOS không?

**Comment by tuetran0795** [comment]
https://developers.momoapp.vn/user/download
@trancongdung12 bạn vào link này để lấy bảng build nha

**Comment by trancongdung12** [comment]
@tuetran0795  nó ra mã qrcode mà máy ảo ios thì k tải được :(

**Comment by tuetran0795** [comment]
bạn chờ mình xem còn bảng build nào không nha

**Comment by tuetran0795** [comment]
@trancongdung12 
hi bạn,
hiện tại thì bên mình chưa support cho enduser cài đặt Momo trên máy ảo nha,
bạn có thể dùng máy thật để tải bản test.

thanks...

**Comment by hunglv156** [comment]
@tuetran0795 tue
Mình đang gặp vấn đề khi test trên thiết bị iOS thì nó k chuyển hướng app test mà toàn chuyển hướng sang app store, có cách nào khắc phục k ạ :(

**Comment by tuetran0795** [comment]
hi @hunglv156 
bạn có thể share file info.plist, code push data không?

**Comment by hunglv156** [comment]
@tuetran0795 tue
<img width="1206" alt="Screen Shot 2022-06-24 at 16 38 16" src="https://user-images.githubusercontent.com/68507984/175508919-1bee44af-e405-4072-8180-be3bc6cc7c69.png">
Mình đang gặp lỗi này khi config bước 2,3 trên ios ạ @@

**Comment by hunglv156** [comment]
Đây ạ
<img width="479" alt="Screen Shot 2022-06-24 at 16 43 33" src="https://user-images.githubusercontent.com/68507984/175509742-4b852d83-9a72-444c-8a92-c1ed00ab6081.png">
<img width="829" alt="Screen Shot 2022-06-24 at 16 43 54" src="https://user-images.githubusercontent.com/68507984/175509745-dd23eda7-dc73-43e2-8370-f81ef768bf7e.png">

**Comment by hunglv156** [comment]
Mình đã mở được vào app test tuy nhiên khi thanh toán xong đang không trừ tiền và k back lại app ạ @@

**Comment by tuetran0795** [comment]
@hunglv156 
Khi thanh toán xong -> app trả kết quả giao dịch như thế nào, có thể cho mình xin số đt bạn dưới test không, khoảng thời gian bạn test.

**Comment by hunglv156** [comment]
@tuetran0795 tue
0963850519
App k trả ra kết quả giao dịch, nó loading rồi ra màn home Momo ạ

**Comment by tuetran0795** [comment]
@hunglv156
bạn thử lại sau nha, mình kiểm tra thấy bạn đang call thanh toán bị failed nên không đá app của mình

**Comment by hunglv156** [comment]
@tuetran0795 
vâng, mình có check lại rồi, do mình chưa thêm appScheme vào info.plist @@. Bạn cho mình hỏi sau khi thanh toán xong app của mình trả về response success nhưng ở bên app test momo k trừ tiền ạ?

**Comment by hunglv156** [comment]
@tuetran0795 tu
giúp mình với ạ

**Comment by tuetran0795** [comment]
@hunglv156
sorry bạn, app test bên mình hôm trước có bảo trì phần thanh toán, bạn thử lại xem, ứng dụng momo có bị trừ tiền không nha??

**Comment by hunglv156** [comment]
@tuetran0795 có thông báo success trả về app nhưng app momo test và production đều k trừ tiền ạ :(

**Comment by hunglv156** [comment]
![Screen Shot 2022-06-28 at 11 34 21](https://user-images.githubusercontent.com/68507984/176093624-415c4760-9490-403c-876f-925dc918dfce.png)

**Comment by trancongdung12** [comment]
@hunglv156  bạn gọi api pay/app chưa? nếu chưa thì nó chưa trừ

**Comment by trancongdung12** [comment]
https://developers.momo.vn/v2/#/docs/app_in_app

**Comment by hunglv156** [comment]
@trancongdung12 ohh, vậy sdk này dành cho phiên bản cũ à bro :(

**Comment by trancongdung12** [comment]
@hunglv156 đúng r. v3 chưa có doc về sdk nha

**Comment by hunglv156** [comment]
@trancongdung12 căn bản app tôi đang có cả bên web mà bên web dùng api mới nhất, còn bên app  dùng sdk thì có sợ nó xảy ra không đồng bộ không? bên app có thể tự thanh toán đc còn hoàn thì gọi api nhờ bên BE xử lý @@

**Comment by tuetran0795** [comment]
@hunglv156 
cái này response về đang đúng rồi nha bạn,
để bên mình check lại rồi note lại cho bạn

**Comment by hunglv156** [comment]
@tuetran0795  cảm ơn bạn, bạn check luôn giúp mình nhé vì mình bị gịuc quá trời luôn r :((

**Comment by thuantvDr4** [comment]
@tuetran0795 
mình mở được app-test nhưng sau thanh toán thì nhận response như này
``
------addListener:failed  {"message": "Parameters valid - ready to open MoMo app. Make sure the application have an implement of [RNMomosdk handleOpenUrl] to receive data", "status": 1}
``
lỗi này cụ thể là bị gì vậy ạ? xin giúp đỡ

**Comment by trancongdung12** [comment]
@tuetran0795 chào bạn.
Mình đã có doc v3 cho dùng sdk chưa ạ

**Comment by thuantvDr4** [comment]
hiện tại thì mình chưa có, mình chỉ làm theo docs trên public thôi
rất mong hỗ trợ

**Comment by Roasted-Peanut** [comment]
@hunglv156 https://github.com/momo-wallet/mobile-sdk/issues/66#issuecomment-1165397686
b fix được lỗi này chưa, mình đang gặp lỗi này

**Comment by Roasted-Peanut** [comment]
> @hunglv156 [#66 (comment)](https://github.com/momo-wallet/mobile-sdk/issues/66#issuecomment-1165397686) b fix được lỗi này chưa, mình đang gặp lỗi này

mình đã fix được.
https://github.com/joltup/rn-fetch-blob/issues/461#issuecomment-553355501
